### DISCUSSION TITLE
[Feature] Forward in time taken ordering support

### DISCUSSION THREAD

**Discussion by dsm1212** [immich-app/immich#7489] [Feature Request]
### The feature

I'd like to be able to show my timeline starting with oldest dated-taken photos (sort forward). And I'd like the slideshow to have a forward in time option. If I photograph an event I reallly want to see it start to finish. Of the two, the slideshow fix is proably the most important, looking at an event backwards is pretty annoying.

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by Menghini** [comment]
I agree with your feature. Moreover, it would be ideal to have an option when sharing albums (and even just viewing albums) to sort in chronological order they were taken, rather than show most recent first.

**Reply by sigulete** [reply]
Yes, this is an option I was looking for as well. 
In particular, within Albums.

**Comment by benmccann** [comment]
There's also a request for a setting at the user level: https://github.com/immich-app/immich/discussions/8068

**Reply by dsm1212** [reply]
I could live with that if you want to close this one.

**Comment by amadejpapez** [comment]
Bumping this as this sorting seems to have been added to albums but not to main home timeline. I just set up Immich and it is really messing up with my mental model as Apple Photos starts at the bottom and goes up and Immich is the exact reverse.

**Reply by JusByte** [reply]
Yes this! I’m used to how Apple photos works with newest at the bottom, and you scroll up for older photos. A toggle for something like this would be great.
